requirements.txt

```diff
@@ -1,4 +1,4 @@
 qiskit>=0.15
-PennyLane>=0.7.0
+git+git://github.com/XanaduAI/pennylane.git#egg=pennylane
```

Comment:
Curious why this is needed

Reply:
A recent PR in PennyLane master renamed the `Variable` class to `VariableRef` --- I wanted to double check that this PR passes against the latest master of PennyLane. However, this renaming will likely be reverted due to side effects that occurred, so this change here is temporary and should be reverted before this PR is merged.